### DISCUSSION TITLE
Symfony 6.0.6 missing asset manifest file

### DISCUSSION THREAD

**Discussion by GrossmanPro** [symfony/symfony#45754] [Symfony Book: The Fast Track] [answered]
I'm trying to learn Symfony by following the tutorial in "Symfony: The Fast Track" on Symfony's site.
All was going well until I got the following error after configuring the EasyAdmin 4 bundle:

> An exception has been thrown during the rendering of a template
> ("Asset manifest file
> "C:\Users\XXX\guestbook/public/build/manifest.json" does not exist.").

Sure enough, it does not exist.
I assume this file should've been generated automaticaly like so much else with Symfony, but if it wasn't created, how do I create it?  
I'm literally just a few pages into this and have no prior experience with PHP frameworks, so I don't have the knowledge to troubleshoot this error.  If there's any further information I should include to assist with solving the problem, please let me know.    Thanks for any help you can provide!

**Comment by norkunas** [comment, marked as answer]
manifest.json file is created not by Symfony, but by webpack. to generate it either run `npm run build` or `npm run watch`

**Reply by GrossmanPro** [reply]
Thank you!  I ran it from the location where the missing manifest file should have been and the admin page started working.
(Pro tip:  Don't forget the step to install node.js!  That was my fault, not the book's.)

**Comment by mariojackson** [comment]
I just ran into the same problem and found this answer. I'd suggest adding the solution to the book?

**Comment by haxworx** [comment]
`composer require symfony/webpack-encore-bundle
npm run build
`
I've been following the book up to this point and was tripped up the same as the above. I managed to get it working, but perhaps like suggested this needs to be resolved in the book content before this stage to avoid this issue?

**Comment by KENNEDOUANLA** [comment]
if you have this error "encore: command not found" after running "npm run watch"  , run "npm install" before "npm run watch"

**Comment by Bousbihfatouma** [comment]
bonjour a tous , merci pour ces infos moi c'est vraiment incroyable j'ai fais toutes les commandes le public/build ne ce créer pas automatiquement . du coup mon site est actuellemnt bloquer avec ce message erreur "An exception has been thrown during the rendering of a template ("Asset manifest file "C:\wamp64\www\illustretonart/public/build/manifest.json" does not exist. Did you forget to build the assets with npm or yarn?").  alors que j'ai fais toute les commandes dans le terminal
